### ISSUE TITLE
Grid system nesting does not work as expected

### ISSUE THREAD

**Issue by alroman** [ucla-oarc-web/WebBlocks#329]
I searched previous issues and I didn't see this come up.

I'm working with the default grid values `structure-panels = 12` and I'm trying out nested panels.  This is an example of the structure I'm building:

``` html
<div class="row">
  <div class="panel-6">
     <div class="row">
       <div class="panel-6">
       </div>
       <div class="panel-6">
         <!-- this column will collapse! -->
       </div>
   </div>
  </div>
  <div class="panel-6">
     <div class="row">
         <div class="panel-12">
           <!-- this column behaves incorrectly -->
         </div>
      </div>
  </div>
</div>
```

I'm expecting the nested panels to create a new 12 column grid where it would be possible to have two `panel-6` columns.  What I get is the second `panel-6` column collapsing.

**Comment by ebollens** [comment]
I just took the following code and laid it into a blank page powered by WebBlocks to test:

``` html
<div class="row">
  <div class="panel-6">
     <div class="row">
       <div class="panel-6 primary">
           A
       </div>
       <div class="panel-6 secondary">
         B
       </div>
   </div>
  </div>
  <div class="panel-6">
     <div class="row">
         <div class="panel-12 tertiary">
           C
         </div>
      </div>
  </div>
</div>
```

Above `$breakpoint-small`, I see the following:

```
[-a-][-b-][---c---]
```

Meanwhile, below `$breakpoint-small`, I see:

```
[---------a--------]
[---------b--------]
[---------c--------]
```

Are you seeing something different? If so, can you link me to the page?

**Comment by alroman** [comment]
After some more testing I've traced the issue to Bootstrap's use of `box-sizing` property; let me explain.  In my brief example, I failed to mention that I was applying a another style directly to the `div` elements with `panel-n` classes.  More accurately, my example should look like this:

``` html
<div class="row">
  <div class="panel-6">
     <div class="row">
       <div class="panel-6 frontpage-box">
       </div>
       <div class="panel-6 frontpage-box">
         <!-- this column will collapse! -->
       </div>
   </div>
  </div>
  <div class="panel-6">
     <div class="row">
         <div class="panel-12 frontpage-box">
           <!-- this column behaves incorrectly -->
         </div>
      </div>
  </div>
</div>
```

My `frontpage-box` class applies a border radius of 1px.  Bootstrap uses `box-sizing: border-box` using this rule:

``` css
.row-fluid [class*="span"] {
/* using ff */
-moz-box-sizing: border-box;
...
}
```

When I tried my example with Bootstrap's responsive grid, my columns collapse accordingly, however when I switched to WebBlocks' grid, I experience the unexpected behavior previously reported.  Removing the 1px border, I get the expected behavior from WebBlocks' grid.

At this point I would suggest having the same `box-sizing: border-box` rule applied to WebBlocks' grid. 

Ref: https://developer.mozilla.org/en-US/docs/CSS/box-sizing
